mixins/localize-mixin.js

```diff
@@ -172,7 +172,9 @@ export const LocalizeMixin = dedupeMixin(superclass => class extends superclass
 		}
 		// eslint-disable-next-line no-prototype-builtins
 		if (this.hasOwnProperty('getLocalizeResources') || this.hasOwnProperty('resources')) {
-			const res = this.getLocalizeResources([...possibleLanguages], config);
+			let superConfig = config;
+			if (superCtor.localizeConfig && superCtor.localizeConfig.importFunc) superConfig = superCtor.localizeConfig;
```

Comment:
The issue I'm seeing with this is that if you reverse the order to be `class extends test2LocalizeDynamicMixn(test`LocalizeDynamicMixn((LitElement)))` then it starts failing.

Reply:
Interesting - I don't think I'm seeing that locally but will dig some more shortly

Reply:
The setup I have is slightly different -- I modified the existing `LocalizeTest` so that it does:

```
// this fails
class LocalizeTest extends LocalizeCoreElement(LocalizeDynamicMixin(LitElement)) {

// this is OK
class LocalizeTest extends LocalizeDynamicMixin(LocalizeCoreElement(LitElement)) {
```

Reply:
So far my finding is that if whatever is using `LocalizeDynamicMixin` is in a separate mixin then it passes in both orders, but when it's in the same file (which is a totally possible usecase) then it doesn't pass when it's in the first order🤔

Reply:
Ok, so after really digging into this and a lot of `console.log`-ing, I think your original solution was correct.

The order of the inheritance chain matters -- it needs to go `LocalizeTest` -> `LocalizeDynamicMixin` (so dynamic mixin can access localize-test's `localizeConfig) -> `LocalizeCoreElement`. The reverse order just isn't going to work, and I think that's OK.